src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -455,4 +457,20 @@ public Charset getEncoding() {
     public void setEncoding(Charset encoding) {
         this.encoding = Objects.requireNonNull(encoding);
     }
+
+    public void clearMetaData() {
+        metaData.clear();
+    }
+
+    public Map<String, List<String>> getMetaData() {
```

Comment:
As I said above, we are trying to get away from the map. So please look for another solution. There is already `getAsStringMap` which is probably what you want.